packages/lib/services/search/SearchEngine.ts

```diff
@@ -544,6 +544,10 @@ export default class SearchEngine {
 
 		let allTerms: Term[] = [];
 
+		if (query.length && query[0] === '/') {
+			query = query.substr(1);
```

Comment:
```suggestion
			query = query.substring(1);
```

Consider using `substring` for new code ([`substr` is deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr)).

Thank you for the pull request!

Reply:
Fixed. Thanks for your suggestion!